src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -190,6 +190,8 @@ internal static class IDEDiagnosticIds
 
         public const string UseNameofInAttributeDiagnosticId = "IDE0280";
 
+        public const string UsePrimaryConstructorDiagnosticId = "IDE0290";
```

Comment:
Just curious: why modern `IDE` diagnostic ids are spread out by 10?

Reply:
It's reasonably common to add more diagnostics to a feature in the future. This allows the ide to keep those later diagnostics associated together. :-)